README.md

```diff
@@ -1,32 +1,32 @@
 # Front-End for multiplayer video game
-An interpretation of Fort Nerf which was created by Bitwise
+<a href="#top"></a>
+An interpretation of Fort Nerf which was created by Bitwise!
 
 # Table of contents
 
 # About the Project
 ## About The Project
-This app build with the existing knowledge of the React basics, and give you further practice working with codebase linting, component testing, Redux Toolkit, and TypeScript. A user should, at the bare minimum, be able to view all posts, add a post, and delete a post. Consider adding the ability to update existing posts, add and set categories on blog posts, specify users, etc.
-This app used Redux Toolkit and state slices to manage any state data needed across multiple components. In the case where data is created and used only within one component (or possibly one and a few of its direct children), consider using useState() or useReducer() and traditional prop drilling. Most apps will use a combination of "local" state management (useState() and useReducer()) and "global" state management (Redux or Context API).
-Apply the TypeScript static typing system. try to use TypeScript for the majority of React components! installed typescript, add support for TypeScript linting, and use either .ts or .tsx extension for the files to apply TypeScript type-checking on.
-Use the `BLANK_README.md` to get started.
+This app build with the existing knowledge of the React basics, and give you further practice working with codebase linting, component testing, Redux Toolkit, and TypeScript. A user should, at the bare minimum, be able to join a game, create a new game, participate with other users in the game, and at the end the game will display the scores of all participants in the game.
 
 ### Built with
 A few of the frameworks/libraries that were used to build the project are
-* [![React][React.js]][React-url]
+* React
+* Typescript
 * Node.js
 * React Redux 
+* Firebase/Firebase Authentication
 * Axios
 * React Router
 * EsLint and AirBnb
 * Phaser
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+* Socket-io Client
+<p align="right">(<a name="top">back to top</a>)</p>
 <!-- ### wireframe
   ### Getting Started The App build out with simple wireframes using Figma.
 <img width="328" alt="wire-frame-frontend" src=""> -->
 
 ## Getting Started
-This is an example of how you may give instructions on setting up your project locally.
-To get a local copy up and running follow these simple example steps.
+To install the FortNerf App Follow these steps
 ### Prerequisites
 * npm
   ```sh
```

Comment:
also on line 39, you got a she instead of sh. not sure how you got away w/ that, but jus wanted to bring it to your attention